_setup_support.py

```diff
@@ -281,7 +294,8 @@ def build_js():
     sys.stdout.flush()
     os.chdir('bokehjs')
 
-    cmd = ["node", "make", 'build', '--emit-error']
+    build_cmd = "build:stripped" if is_release else "build"
+    cmd = ["node", "make", "clean", build_cmd]
```

Comment:
@mattpap  AFIACT `--emit-error` is superfluous now but please let me know if I am mistaken